js/plugins/google-genai/src/googleai/imagen.ts

```diff
@@ -89,6 +89,7 @@ function commonRef(
 ): ModelReference<ConfigSchemaType> {
   return modelRef({
     name: `googleai/${name}`,
+    version: name,
```

Comment:
This should not be necessary. It should work without the version. Version is always optional, and in fact we're working towards making it obsolete. If set, use it as model name, if not it should use whatever is in the ref name.

Reply:
refactored